build_runner/lib/src/asset_graph/node.dart

```diff
@@ -27,8 +27,34 @@ abstract class AssetNode {
   /// exist.
   Digest lastKnownDigest;
 
+  /// Whether or not this node was an output of this build.
+  bool get isGenerated => false;
+
+  /// Whether or not this asset can be read.
```

Comment:
"Whether or not this asset type can be read.

This does not indicate whether the node is readable at this moment in time."

Reply:
done